citizen_science/citizen_science.md

```diff
@@ -0,0 +1,264 @@
+<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>
+
+Check out this recent article exploring the reasons people participated in a citizen science project:
+
+[Why (not) participate in citizen science? Motivational factors and barriers to participate in a citizen science program for malaria control in Rwanda](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0237396)
+
+</div>
+
+As you learn about new citizen science projects, read their recruitment materials and think about what kinds of motivation they're attempting to tap into to attract citizen scientists.
+
+## How to build a citizen science project
+
+Large citizen science projects take a substantial effort to get off the ground but are worthwhile because they pay dividends for a long time once they reach a certain momentum.
+It may take years before a project produces valuable scientific output.
+Successful projects often rely on grant funding to pay for dedicated staff.
+
+For a quicker start, see if there's an existing project you can join rather than starting your own.
+If you can contribute time and expertise to an existing project, it may be possible for you to add tests or experiments to their ongoing work.
+
+<div class = "learn-more">
+<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>
+
+For an idea of what might be involved in setting up a larger project, see the [Federal Crowdsourcing and Citizen Science Toolkit](https://www.citizenscience.gov/toolkit/#) published by the U.S. government.
+
+</div>
+
+A more reasonable option for researchers looking to get started with citizen science is to post a project on an existing platform, like [Zooniverse](https://www.zooniverse.org/lab).
+
+
+## Quiz: Citizen Science
+
+Which of the following are ways citizen scientists can help a project? Check all that apply.
+
+[[X]] data/sample collection
+[[X]] data coding
+[[X]] scientific programming
+[[X]] donating computing power
+***
+<div class = "answer">
+
+These are all examples of how citizens can support scientific discovery!
+
+</div>
+***
+
+Which of the following might motivate someone to participate as a citizen scientist? Check all that apply.
+
+[[X]] learning something interesting
+[[X]] wanting to be involved in science
+[[X]] payment or gift incentives
+[[X]] fun
+[[X]] competitiveness
+[[X]] wanting to give back to their community
+[[X]] wanting support an important cause
+***
+<div class = "answer">
+
+All of these are reasons people put their time and effort towards citizen science projects!
+
+As you learn about new citizen science projects, notice the language they use on their recruitment materials and think about what kinds of motivation they're attempting to tap into.
+If you start your own citizen science project, carefully consider what might best motivate your participants.
+
+</div>
+***
+
+True or False: There are platforms available online to help you launch new citizen science projects quickly.
```

Comment:
I was thrown by the word "quickly" given how the module had early emphasized how these projects can take a long time to get momentum to produce scientific results.

Reply:
Good call! Will revise
```suggestion
True or False: There are platforms available online to help you launch new citizen science projects without having to do all of the setup work yourself.
```

Reply:
Good call --- I revised